railgun/hack/generators/manifests/main.go

```diff
@@ -0,0 +1,157 @@
+package main
```

Comment:
There is a 1-liner alternative to be considered:

`kubectl patch --local -o yaml -f - --patch '{"spec": {"preserveUnknownFields": false}}' --type merge`

Caveats:
- takes 1 file at a time
- doesn't work if the file has more than 1 yaml document within it (but a single document with a leading/trailing `---` seems fine)

Is this alternative feasible? I think that using an existing program is strictly better than writing a new one, where possible and reasonable.

Because of that, I didn't do a full review of this program - will do another round if we opt for keeping it.

Reply:
Yeah let's go with that. closing.